### PR TITLE
doc: remove support for CentOS7

### PR DESCRIPTION
**Pull request by annastuchlik**
This PR removes support for **CentOS 7** from the docs.

The change applies to version 5.4,so must be backported to branch-5.4.

Refs scylladb/scylla-enterprise#3502

In addition, this commit removes the information about Amazon Linux and Oracle Linux, unnecessarily added without request, and there's no clarity over which versions should be documented.

### CODE REVIEW

File: docs/getting-started/os-support.rst [hunk 1]
```diff
@@ -25,16 +25,16 @@ ScyllaDB Open Source
...
-+----------------------------+------+------+-------+-------+---------+-------+-------+-------+------+--------------+
++----------------------------+-------------+---------------+---------------+--------------+--------------+
+| Linux Distributions        |Ubuntu       | Debian        | Rocky /       | Oracle Linux | Amazon Linux |
+|                            |             |               | RHEL          |              |              |
++----------------------------+------+------+-------+-------+-------+-------+-------+------+--------------+
+| ScyllaDB Version / Version |20.04 |22.04 |  10   |  11   |   8   |   9   |  7.6  | 8.1  | 2            |
++============================+======+======+=======+=======+=======+=======+=======+======+==============+
+|   5.4                      | |v|  | |v|  | |v|   | |v|   | |v|   | |v|   | |v|   | |v|  | |v|          |
++----------------------------+------+------+-------+-------+-------+-------+-------+------+--------------+
+|   5.2                      | |v|  | |v|  | |v|   | |v|   | |v|   | |x|   | |x|   | |x|  | |x|          |
```
Comment (yaronkaikov) [inline docs/getting-started/os-support.rst:36]: @annastuchlik Centos7 support was removed only from 5.4, in 5.2 we still support it

**Review by yaronkaikov — CHANGES_REQUESTED** [review]

File: docs/getting-started/os-support.rst (hunk 1 above)
Comment (yaronkaikov) [reply docs/getting-started/os-support.rst:36]: And also oel76 was removed from 5.4 only

File: docs/getting-started/os-support.rst [hunk 2]
```diff
@@ -28,13 +28,13 @@ ScyllaDB Open Source
...
 |                            |             |               | RHEL    | RHEL          |              |              |
-+----------------------------+------+------+-------+-------+---------+-------+-------+-------+------+--------------+
-| ScyllaDB Version / Version |20.04 |22.04 |  10   |  11   | 7       |   8   |   9   |  7.6  | 8.1  | 2            |
-+============================+======+======+=======+=======+=========+=======+=======+=======+======+==============+
-|   5.4                      | |v|  | |v|  | |v|   | |v|   | |v|     | |v|   | |v|   | |v|   | |v|  | |v|          |
-+----------------------------+------+------+-------+-------+---------+-------+-------+-------+------+--------------+
-|   5.2                      | |v|  | |v|  | |v|   | |v|   | |v|     | |v|   | |x|   | |x|   | |x|  | |x|          |
-+----------------------------+------+------+-------+-------+---------+-------+-------+-------+------+--------------+
++----------------------------+------+------+-------+-------+---------+-------+-------+--------------+--------------+
+| ScyllaDB Version / Version |20.04 |22.04 |  10   |  11   |   7     |   8   |   9   | 8.1          | 2            |
```
Comment (mykaul) [inline docs/getting-started/os-support.rst:32]: I find it strange we have OEL 8.1 here, which is quite ancient (8.8! is the latest - https://docs.oracle.com/en-us/iaas/images/oracle-linux-8x/ )

Comment (avikivity) [reply docs/getting-started/os-support.rst:32]: I don't remember that we removed CentOS 7 support from 5.4.

Comment (mykaul) [reply docs/getting-started/os-support.rst:32]: Sent you the reference for this decision (in a separate issue)

**Review by tzach — APPROVED** [review]

**Comment by denesb** [comment]
Backported to 5.4.

**Pull request closed without merging**